### PR TITLE
Feather icon requests

### PR DESCRIPTION
**Pull request by Andreto**
Adds FEATHER_ICON_REQUESTS.md which contains a table with open Feather icon requests that have not been implemented in Lucide and do not contain any branded logos. I put the file in the docs folder but maybe it should be placed somewhere else.

I also added a section in the contribution guidelines about the Feather icon requests.

### CODE REVIEW

**Review by locness3 — APPROVED** [review]

**Comment by ericfennis** [comment]
Awesome work! Nice to have this overview. 
I'm having a bit of doubts if this need to be living in the code base or maybe better to have an issue for this.
🤔

**Comment by Andreto** [comment]
@ericfennis You are probably right. Would be easier to update and maintain that way. Maybe still a link to it in the contribution guidelines?

**Comment by ericfennis** [comment]
I think also nice that you can also reference the issue when making a PR. 
yes keep the link in the contributions.

**Comment by Andreto** [comment]
I'll create an issue and revert the FEATHER_ICON_REQUESTS.md commits in this PR.

**Comment by Andreto** [comment]
That should be it.
#119

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -22,3 +22,6 @@ Guidelines for pull requests:
 Before creating an icon request, please search to see if someone has requested the icon already. If there is an open request, please add a :+1:.
 
 If the icon has not already been requested, [create an issue](https://github.com/lucide-icons/lucide/issues/new?title=Icon%20Request:) with a title of `Icon request: <icon name>` and add as much information as possible.
+
+## Icon Requests from Feather
+If you are a designer who wants to contribute to Lucide but you don't know what icons to work on, then have a look at the Requests from Feather. All open, unfinished and valid requests can be found in [Fether Icon Requests](https://github.com/lucide-icons/lucide/issues/119).
```
Comment (ericfennis) [inline CONTRIBUTING.md:27]:
```suggestion
If you are a designer who wants to contribute to Lucide but you don't know what icons to work on, then have a look at the Requests from Feather. All open, unfinished and valid requests can be found in [Feaher Icon Requests](https://github.com/lucide-icons/lucide/issues/119).
```

Comment (Andreto) [reply CONTRIBUTING.md:27]: Maybe I'm blind but I don't see it, could you fill me in?

Comment (Andreto) [reply CONTRIBUTING.md:27]: Ok, got it now.

Comment (ericfennis) [reply CONTRIBUTING.md:27]: You wrote `fether` but it is `feather`

**Comment by omar-abdelatif** [comment]
awesome work, but i have an idea which is to add some icons about islamic and christian graves, tombs icons. thanks in advance

**Pull request merged**